cmd/res/configuration.toml

```diff
@@ -0,0 +1,35 @@
+[Service]
```

Comment:
as mentioned on main.go, I think the location of this file should be `/cmd/edgex-ui-server/res/configuration.toml`